src/crawlers/browser_crawler.js

```diff
@@ -285,7 +285,7 @@ class BrowserCrawler extends BasicCrawler {
         this.postNavigationHooks = postNavigationHooks;
 
         if (useSessionPool) {
-            this.persistCookiesPerSession = persistCookiesPerSession || true;
+            this.persistCookiesPerSession = persistCookiesPerSession;
```

Comment:
Since we're changing the defaults, I would move this outside of the `if` clause. So that it's always a proper boolean and not sometimes `undefined`.

Reply:
Kept inside `if` clause, which would set to `true`, and also added `else` which would set it to `false`.